### PR TITLE
BHPA privacy page

### PR DESCRIPTION
**Pull request by mattdelsordo**
1. Added privacy page
2. Hunted down and cleaned up a bunch of console errors

### CODE REVIEW

File: frontend/front/src/pages/Public/Layout/Navbar.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ function Navbar(props) {
 
   const open = Boolean(anchorMore);
 
-  const handleClickMore = (event) => setAnchorMore(event.currentTarget);
+  const handleClickMore = (event) => setAnchorMore(event?.currentTarget);
```
Comment (thiagobardini) [inline frontend/front/src/pages/Public/Layout/Navbar.js:75]: Hey @mattdelsordo, you need to undo this part. The "Learn More" dropdown menu (desktop view) is unclickable.

**Review by thiagobardini — CHANGES_REQUESTED** [review]
Looks good! We just need to fix the code for the dropdown menu (navbar) to make it clickable again. Thanks for reviewing my code too!

File: frontend/front/src/pages/Public/Layout/Navbar.js (hunk 1 above)
Comment (mattdelsordo) [reply frontend/front/src/pages/Public/Layout/Navbar.js:75]: Ack, sorry! Should have tested this harder!

Comment (mattdelsordo) [reply frontend/front/src/pages/Public/Layout/Navbar.js]: Should be fixed now

**Review by thiagobardini — APPROVED** [review]

**Pull request merged**